### PR TITLE
[prebuild] Make prebuild logs view more responsive

### LINKED ISSUE #11189: Prebuild logs button hidden from view
[[Internal context](https://gitpod.slack.com/archives/C02SF4A050W/p1656942542481819)]

### PR DESCRIPTION
**Pull request by laushinka**
## Description
This makes sure the buttons will always show (and view always scrollable) regardless the screen size.
Tested on workspace start, prebuild detail view, and prebuild prefix trigger.

Screenshot 2022-07-07 at 22 41 04
Screenshot 2022-07-07 at 22 40 09

## Related Issue(s)

Fixes #11189

## How to test
1. Make a change to a repo with a prebuild
2. Resize the screen to a small size, buttons should always be in view.

## Release Notes

```release-note
Make prebuild logs responsive for small viewports
```

## Werft options:

- [x] /werft with-preview

### CODE REVIEW

File: components/dashboard/src/components/PrebuildLogs.tsx
Comment (gtsiolis) [inline components/dashboard/src/components/PrebuildLogs.tsx:147]: **issue**: This is probably affecting the position of the bottom part and the alignment.

| Prebuild | Prebuild in Progress |
|-|-|
| prebuild-1 | prebuild-2 |

**Review by gtsiolis — COMMENTED** [review]
Thanks for taking action and opening this, @laushinka! 

Overall direction looks good, but this introduces a regression affecting the layout and alignment of the bottom part. Not sure if we'd like to merge this before fixing this. ❗

File: components/dashboard/src/components/PrebuildLogs.tsx
Comment (laushinka) [reply components/dashboard/src/components/PrebuildLogs.tsx]: Thanks for the quick help, @gtsiolis 🙏🏽 I'm back now and will follow-up. Build is running for a very long time now..

Comment (laushinka) [reply components/dashboard/src/components/PrebuildLogs.tsx]: Blocked by failing previews. Switching priority for now as talked about [here](https://gitpod.slack.com/archives/C02EN94AEPL/p1657178558353579?thread_ts=1657178158.494489&cid=C02EN94AEPL).

**Comment by laushinka** [comment]
@gtsiolis Cut-off logs should be fixed now - screenshots in the main description 👆🏼

**Review by geropl — APPROVED** [review]
Tested, and works great now! :tada: 

/block in case someone else is meant to (re-)review. Leaving it up to @laushinka to go ahead!

**Review by gtsiolis — APPROVED** [review]
UX LGTM, @laushinka! ✨ 

Thanks @geropl for holding this for one last UX review. 🙏 

Removing also the hold to save time for anyone else, since this is a trivial fix. 🏁

**Pull request merged**
